File: src/com/jda/algorithm_programs/DayOfWeek.java
package com.jda.algorithm_programs;

import com.jda.utility.Util;
import com.jda.utility.Utility;
import com.jda.utility.Utility.Reader;

public
class DayOfWeek {

  public static
  void main(String[] args) {
    System.out.println("Enter date in \"mm dd yyyy\" format");
    Reader reader = new Utility().new Reader();
    int mm = reader.readInteger();
    int dd = reader.readInteger();
    int yyyy = reader.readInteger();
    System.out.println(Util.dayOfWeek(mm, dd, yyyy));
  }
}

File: src/com/jda/algorithm_programs/DayOfWeek.kt
package com.jda.algorithm_programs

import com.jda.utility.Util
import com.jda.utility.Utility

object DayOfWeek {

    @JvmStatic
    fun main(args: Array<String>) {
        println("Enter date in \"mm dd yyyy\" format")
        val reader = Utility().Reader()
        val mm = reader.readInteger()
        val dd = reader.readInteger()
        val yyyy = reader.readInteger()
        println(Util.dayOfWeek(mm, dd, yyyy))
    }
}
